rules/build_sector.smk

```diff
@@ -1310,6 +1318,36 @@ rule build_shipping_demand:
         "../scripts/build_shipping_demand.py"
 
 
+if MOBILITY_PROFILES_DATASET["source"] in ["build"]:
+
+    rule build_mobility_profiles:
+        params:
+            sector=config_provider("sector"),
+        input:
+            zip_files=storage(
+                expand(
+                    MOBILITY_PROFILES_DATASET["url"],
+                    year=[2010, 2011, 2012, 2013, 2014],
+                    street_type=["A", "B"],
+                ),
+            ),
```

Comment:
Wondering if the raw input downloads for `build` rules should also be isolated in `retrieve` rules. In the past, one problem with the `storage()` function was that it would trigger re-runs unnecessarily.

Reply:
I suggest we observe the situation if problems re-occur again now. I would assume in most cases that `archive` is the default that people use.
For consistency with the conventions in this PR, if the primary data for a `rule build_*` is retrieved, then setting the `source: archive` should also only retrieve the primary data and pass it on to the `rule build_*`. But that is just a detail.